### PR TITLE
three-dc-local-read: set tidb_replica_read as closest replicas

### PR DESCRIPTION
**Pull request by hehechen**
### What is changed, added or deleted? (Required)
Update three-dc-local-read.md.

After https://github.com/pingcap/tidb/pull/28033/files merged, need to set tidb_replica_read as closest-replicas for local read.
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: best-practices/three-dc-local-read.md [hunk 1]
```diff
@@ -22,6 +22,6 @@ zone=dc-1
 
 [Stale Read](/stale-read.md) 为用户提供了一种读取历史数据的一种机制。使用 Stale Read 功能，你能从指定时间点或时间范围内读取对应的历史数据，从而避免数据同步带来延迟。在部分跨数据中心部署的场景中使用 Stale Read 功能，通过牺牲一定的实时性，TiDB 可就近访问对应数据所在当前中心的副本，避免跨数据中心的网络延迟，降低整体查询的访问延迟。
 
-当 TiDB 收到 Stale Read 查询时，假如对应的 TiDB 配置了 `zone` 标签，就会将请求发送到对应数据副本所在 TiKV 拥有相同的 `zone` 标签的节点上。
+当 TiDB 收到 Stale Read 查询时，假如对应的 TiDB 配置了 `zone` 标签，而且将 `tidb_replica_read` 设置为 `closest-replicas`，就会将请求发送到对应数据副本所在 TiKV 拥有相同的 `zone` 标签的节点上。
```
Comment (Oreoxmt) [inline best-practices/three-dc-local-read.md:25]:
```suggestion
当 TiDB 收到 Stale Read 查询时，假如对应的 TiDB 配置了 `zone` 标签，而且 [`tidb_replica_read`](/system-variables.md#tidb_replica_read-从-v40-版本开始引入) 为 `closest-replicas`，就会将请求发送到对应数据副本所在 TiKV 拥有相同的 `zone` 标签的节点上。
```

**Comment by Oreoxmt** [comment]
@hehechen Could you please invite a tech review? Thanks.

**Comment by hehechen** [comment]
> @hehechen Could you please invite a tech review? Thanks.

@Yisaer  PTAL

**Comment by Oreoxmt** [comment]
@Yisaer PTAL

**Comment by hehechen** [comment]
@nolouch PTAL

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
@Yisaer @nolouch PTAL

**Comment by Oreoxmt** [comment]
@Yisaer PTAL

**Review by Yisaer — APPROVED** [review]

**Pull request merged**